heliopy/data/ace.py

```diff
@@ -158,9 +158,52 @@ def swi_h2(starttime, endtime):
     product = 'swi_h2'
     fname = 'h2_swi'
     version = '09'
+    keys = {'C5_qual': 'C5_qual',
```

Comment:
Are all these the same apart from the `Epoch` --> `Time` one? If so it might be worth modifying the `_ace()` method to just copy across all keys, apart from `Epoch` which it can just automatically change to `Time`.